src/modules/markCompleted.js

```diff
@@ -0,0 +1,16 @@
+const updateTasks = (index, taskList) => {
+  taskList.forEach((item) => {
+    if (item.index === index) {
+      item.completed = !item.completed;
+    }
+  });
```

Comment:
Hi there! Ver nice layout and project readability. However, please use descriptive names for your variables to enhance readability. Here, you can use task instead of item.